microservices/ga/datamgr/README.md

```diff
@@ -200,7 +201,7 @@ Help output:
   Choose a command run in films:
 
   compile         Compile the binary.
-  clean           Remove dep, vendor, binary(s), and execute go clean
+  clean           Remove dep, vendor, binary(s), and executes go clean
```

Comment:
@MarkGreenPR did you change this in your branch to modules?

Reply:
Yes, in Makefile it is now:
Remove binary(s) and execute go clean